src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -319,13 +323,13 @@ public override bool Execute ()
 					foreach (var ca in resolver.GetAssembly (assemblyItem.ItemSpec).CustomAttributes) {
 						switch (ca.AttributeType.FullName) {
 						case "Android.IncludeAndroidResourcesFromAttribute":
-							AddAttributeValue (androidResources, ca, "XA5206", "{0}. Android resource directory {1} doesn't exist.", true, fullPath);
+							AddAttributeValue (androidResources, ca, "XA5206", "{0}. (Details: Android resource directory {1} doesn't exist.)", true, fullPath);
```

Comment:
Why the addition of `(Details:` and `)`?

This might make sense if XA5207 were emitted *twice*, and the "(Details:" message was the *second* message,  providing additional information for the first message, but this isn't the case; XA5207 is only emitted once per "logical construct" (assembly/etc.).